graylog2-web-interface/src/components/datanode/Constants.ts

```diff
@@ -137,7 +137,7 @@ export const MIGRATION_STATE = {
   },
 } as const;
 
```

Comment:
ROLLING_UPGRADE_MIGRATION_WELCOME_PAGE: {
         key: 'ROLLING_UPGRADE_MIGRATION_WELCOME_PAGE',
         description: 'Welcome to `Rolling upgrade` migration',
 }